recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xt-log/xt-log_git.bb

```diff
@@ -0,0 +1,12 @@
+DESCRIPTION = "C++ header only log"
```

Comment:
I think this should be split: common goes to meta-xt-images and the rest stays here as bbappend

Reply:
Done